tests/src/whisk/core/controller/test/EntitlementProviderTests.scala

```diff
@@ -42,7 +44,9 @@ import whisk.core.entity._
  * "using Specs2RouteTest DSL to chain HTTP requests for unit testing, as in ~>"
  */
 @RunWith(classOf[JUnitRunner])
-class EntitlementProviderTests extends ControllerTestCommon {
+class EntitlementProviderTests
+    extends ControllerTestCommon
+    with ScalaFutures {
```

Comment:
What are you using this trait for? Note that it gives you even simpler convenience methods like

`check.futureValue shouldBe expected`